OxfordBodleian/Juel-Jensen/BDLaethe39.xml

```diff
@@ -205,37 +213,49 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                     </dimensions>
                                 </layout>
                             </layoutDesc>
-                           
+
                         </objectDesc>
-                        
+
                         <handDesc>
                             <handNote xml:id="h1" script="Ethiopic">
-                           
+
                             </handNote>
                         </handDesc>
-                        
+
                        <!-- <decoDesc>
-                         
+
                         </decoDesc>-->
-                        
-                     <!--   <bindingDesc>
-                            <binding xml:id="binding">
-                                <decoNote xml:id="b1"></decoNote>
-                                <decoNote xml:id="b2" type="bindingMaterial">
-                           <material key="wood"/>
-                        </decoNote>
+
+                        <bindingDesc>
+                            <binding contemporary="Ethiopian false">
```

Comment:
by "false" I understand here that the binding is not the original one. Is it here the case that the entire binding has been replaced, including the boards, or that elements (the sewing) have been repaired? I wonder whether it might make sense to make this explicit in a note? Maybe my question does not make conceptual sense.

Reply:
I think that the boards may be contemporary with the content but the sewing has been repaired certainly after, when the strips of textile and parchment have been placed around the quires. You are right, it should be specified. Which is the best place for the note in your opinion? After @contemporary?